.prettierignore

```diff
@@ -0,0 +1,2 @@
+/package.json
+/yarn.lock
```

Comment:
yarn.lock is not being formatted anyway by Prettie.. but I don't agree that `package.json` is autogenerated..